content/projects/tdd/string-calculator/_index.md

```diff
@@ -209,10 +209,10 @@ add("1,2,3//;\n1000,1;2")
 
 ## Instructions For Reviewers
 
-- Look out for file structure. It might seem obvious, but people sometimes mess up the file structure of a TDD project. Refer to : **Set up your environment** which is at the top of this page.
+- Tests are nessesary for this project. Proper testing to be followed. If someone uses mocks and spies - that is incompetent.
 
-- Code should be clean, DRY and should follow the KISS(Keep It Simple, Stupid) principal - look out for these when reviewing. Also ensure that they have used a tool like Prettier or Black to format their code.
+- Make sure that the output is as per project instructions. Proper error handling to be used. no `print`, `console.log`. an error is to be thrown when needed and a value returned when needed.
```

Comment:
Remove this piece: `Make sure that the output is as per project instructions.` as I think that is a given for any project :)